lib/travis_event/handlers/stalled_finished_job.rb

```diff
@@ -0,0 +1,114 @@
+require 'travis/client'
+
+module TravisEvent
+  module Handlers
+    class StalledFinishedJob
```

Comment:
Sidekiq workers should live in the workers directory...that's where they are expected to live